src/main/java/org/mandas/docker/client/DefaultDockerClient.java

```diff
@@ -969,7 +967,7 @@ public TopResults topContainer(final String containerId, final String psArgs)
       throws DockerException, InterruptedException {
     try {
       WebTarget resource = resource().path("containers").path(containerId).path("top");
-      if (!Strings.isNullOrEmpty(psArgs)) {
+      if (psArgs != null && "".equals(psArgs.trim())) {
```

Comment:
Shouldn't this be checking that psArgs is NOT empty?

Reply:
good catch